src/DotVVM.Framework/Utils/ReflectionUtils.cs

```diff
@@ -382,6 +383,17 @@ public static Type UnwrapTaskType(this Type type)
                 return type;
         }
 
+        public static Type UnwrapValueOrBinding(this Type type)
```

Comment:
Shall we also support `IValueBinding<T>` and others here?

Reply:
IValueOrBinding<T> does not exist, only the struct ValueOrBinding is generic